research/frontend/css/index.css

```diff
@@ -2,7 +2,12 @@ html,
 body {
   width: 100%;
```

Comment:
Usually it's good to list the css properties in lexicographical order (alphabetical order in our case)